contracts/base/Settings.sol

```diff
@@ -359,6 +361,25 @@ contract Settings is SettingsInterface, TInitializable, Pausable, BaseUpgradeabl
         );
     }
 
+    /**
+        @notice It sets the cToken address for a specific asset address.
+        @param assetAddress asset address to configure.
+        @param cTokenAddress the new cToken address to configure.
+     */
+    function _setCTokenAddress(address assetAddress, address cTokenAddress) internal {
+        if (assetAddress == ETH_ADDRESS) {
+            cTokenAddress.requireEqualTo(0x4Ddc2D193948926D02f9B1fE9e1daa0718270ED5, "CETH_ADDRESS_NOT_MATCH");
```

Comment:
This is hardcoded because the cETH token contract does not have an `underlying()` function. So there is no way to check if the Compound contract that we are passing is actually valid for ETH. 

If you can think of a better option for this, I'm open to it. Also I can't imagine there is a point in which we will need to update the cToken addresses in the asset settings. Unless something catastrophic happens to Compound that requires them to deploy new cToken contracts for an asset they shouldn't ever be changed honestly. This could be a potential security risk if one of the "pausers" calls the `updateCTokenAddress` function that is for a malicious contract.

Reply:
Okay, in the meantime, could we add a constant with a comment explaining this?